From 82c6b1a22f26cfe8fe867527cbe4e444f2df7afa Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Tue, 27 Nov 2018 15:48:09 +0800
Subject: [PATCH 5210/5242] MLK-20471-2: ARM64: dtsi: imx8qm-hdmi: Move
 sai_hdmi_rx to right file

commit  7f31a41baebdbba725fe48f9d9849c4ee9aa2daa from
https://source.codeaurora.org/external/imx/linux-imx.git

Move sai_hdmi_rx to fsl-imx8qm-mek-hdmi-in.dts.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
(cherry picked from commit f9e95e95796830ed6c22b95f96be9370461796fe)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-mek-hdmi-in.dts  |    5 +++++
 .../boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts     |    5 -----
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi-in.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi-in.dts
index 427ece6..7886808 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi-in.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi-in.dts
@@ -60,3 +60,8 @@
 	assigned-clock-rates = <0>, <0>, <400000000>;
 	status = "okay";
 };
+
+&sai_hdmi_rx {
+	fsl,sai-asynchronous;
+	status = "okay";
+};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
index 74e5aad..59532f4 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-hdmi.dts
@@ -79,11 +79,6 @@
 	status = "okay";
 };
 
-&sai_hdmi_rx {
-	fsl,sai-asynchronous;
-	status = "okay";
-};
-
 &spdif1 {
 	assigned-clocks =<&clk IMX8QM_AUD_PLL0_DIV>,
 			<&clk IMX8QM_AUD_ACM_AUD_PLL_CLK0_DIV>,
-- 
1.7.9.5

